languages/elixir/concepts/agent/links.json

```
@@ -4,11 +4,11 @@
     "description": "task-module"
   },
   {
-    "url": "https://hexdocs.pm/elixir/Agent.html",
-    "description": "elixir-doc-agent"
+    "url": "https://hexdocs.pm/elixir/GenServer.html",
+    "description": "genserver-module"
```

I think it's more appropriate to link to the Agent module, not to the GenServer module. There will be a GenServer concept of its own at some point.
```suggestion
    "url": "https://hexdocs.pm/elixir/Agent.html",
    "description": "The Agent module"
```